### PR TITLE
Adding Additional Notes To Email To A Friend Functionality

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Adding Additional Notes To Email To A Friend Functionality for storefront

This pull request (PR) ...

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/email-a-friend.html

### CODE REVIEW

File: src/marketing/email-a-friend.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ _Email a Friend_
 
 When this feature is configured, store customers follow these steps to share product information with friends.
 
-1. On a catalog page, click the **Email** link. Then, do one of the following:
+1. On a catalog page, click the **Email** link. Then, if the feature configured only for registered users, do one of the following:
```
Comment (dmytro-ch) [inline src/marketing/email-a-friend.md:44]:
```suggestion
1. On a catalog page, click the **Email** link. Then, if the feature is configured only for registered users, do one of the following:
```

File: src/marketing/email-a-friend.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ When this feature is configured, store customers follow these steps to share pro
     - Enter the **Name** and **Email Address** of the additional person.
 
         You can send the message to as many additional people as the configuration allows.
+        
+         You can remove added Invitee by clicking the **Remove** link, near **Email** field of needed Invitee.
```
Comment (dmytro-ch) [inline src/marketing/email-a-friend.md:57]:
```suggestion
        You can remove the added Invitee by clicking the **Remove** link, near **Email** field of needed Invitee.
```

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

File: src/marketing/email-a-friend.md [hunk 3]
```diff
@@ -53,6 +53,7 @@ When this feature is configured, store customers follow these steps to share pro
     - Enter the **Name** and **Email Address** of the additional person.
 
         You can send the message to as many additional people as the configuration allows.
+        You can remove the added Invitee by clicking the **Remove** link, near **Email** field of needed Invitee.
```
Comment (leslietilling) [inline src/marketing/email-a-friend.md:56]:
```suggestion
        You can remove the added invitee by clicking the **Remove** link.
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**